phoenix-core/src/main/java/org/apache/phoenix/iterate/ChunkedResultIterator.java

```diff
@@ -74,7 +74,7 @@ public ChunkedResultIteratorFactory(ParallelIteratorFactory
 
         @Override
         public PeekingResultIterator newIterator(StatementContext context, ResultIterator scanner, Scan scan, String tableName) throws SQLException {
-            if (logger.isDebugEnabled()) logger.debug(LogUtil.addCustomAnnotations("ChunkedResultIteratorFactory.newIterator over " + tableRef.getTable().getName().getString() + " with " + scan, ScanUtil.getCustomAnnotations(scan)));
+            if (logger.isDebugEnabled()) logger.debug(LogUtil.addCustomAnnotations("ChunkedResultIteratorFactory.newIterator over " + tableRef.getTable().getPhysicalName().getString() + " with " + scan, ScanUtil.getCustomAnnotations(scan)));
```

Comment:
https://issues.apache.org/jira/browse/PHOENIX-2807
Created a jira for the same for now.